libs/application/templates/rental-agreement/src/fields/PropertySearch/index.tsx

```
@@ -0,0 +1,482 @@
+                                                unit,
+                                                Number(e.target.value),
+                                              )
+                                            }
+                                            unitInputErrorMessage={
+                                              errors?.registerProperty?.[
+                                                `searchResults.units`
+                                              ]
+                                            }
+                                          />
+                                        )
+                                      })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)